### DISCUSSION TITLE
warning with the gpg key

### DISCUSSION THREAD

**Discussion by Karduin** [vkbo/novelWriter#1157] [General]
Linux Mint.
it's not a big deal because it's just a warning and install run properly.
But when i add repository with `sudo add-apt-repository ppa:vkbo/novelwriter` as mentioned in the getting started for Ubuntu / Mint, i have a warning with the gpg key.

```
❯ sudo add-apt-repository ppa:vkbo/novelwriter
Vous êtes sur le point d’ajouter le PPA suivant :
 novelWriter is an open source plain text editor designed for writing novels. It supports a minimal markdown syntax for formatting text. It is written with Python 3 (3.6+) and Qt 5 (5.3+) for cross-platform support. See also https://novelwriter.io/
 Plus de renseignements : https://launchpad.net/~vkbo/+archive/ubuntu/novelwriter
Appuyez sur Entrée pour continuer ou Ctrl+C pour annuler

Warning: apt-key is deprecated. Manage keyring files in trusted.gpg.d instead (see apt-key(8)).
Executing: /tmp/apt-key-gpghome.kRmhNgkJNe/gpg.1.sh --keyserver hkps://keyserver.ubuntu.com:443 --recv-keys 5C68F4BDB1AA0A6E66276893F19F1FCE50043114
gpg: clef F19F1FCE50043114 : clef publique « Launchpad PPA for Veronica Berglyd Olsen » importée
gpg: Quantité totale traitée : 1
gpg:               importées : 1
```

Work's fine with Debian procedure.
Kind regards,
Jim

**Comment by vkbo** [comment]
Thanks for the info. The apt-key method has been deprecated for a good while in Debian. Seems the same change has occurred in Mint too now, so I'll update the docs.

Would you mind making an issue ticket with this? You can tag it with the documentation label, and I'll fix it when I update the docs for the next release.
